src/main/java/jimmy/mcgymmy/logic/commands/EditCommand.java

```diff
@@ -24,6 +24,7 @@
  */
 public class EditCommand extends Command {
     public static final String COMMAND_WORD = "edit";
+    public static final String SHORT_DESCRIPTION = "Edit the selected food item.";
```

Comment:
Inconsistent verb tense: In some files, the string is present simple tense whereas, in other files, it is infinitive.